server/include/ServerNetwork.h

```diff
@@ -1,6 +1,6 @@
 #if defined(_WIN32)
 #include <WinSock2.h>
-#include <Windows.h>
+#include <windows.h>
```

Comment:
you used "#include <**W**indows.h>" in ur latest commit in main branch. did u mean to change it back here?

Reply:
was intentional, ide was complaining